environment.rb

```diff
@@ -55,5 +54,4 @@ class Iam < Sinatra::Base
   end
 
   set :DB, Sequel.connect(settings.database) if defined? settings.database
-  set :REDIS, Redis.new(host: ENV['REDIS_HOST']) if defined? ENV['REDIS_HOST']
```

Comment:
Do you think it would be better if we put a `require_relative '../../lib/cache.rb'` in the environment.rb file so it behaves more like the old `@redis`?